openstack/cronus/templates/nebula/_config.yaml.tpl

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.nebula.enabled -}}
+nebula:
+  cacheSize: {{ .Values.nebula.cacheSize }}
+  listenAddr:
+    http: # default :1080
+  keystone:
```

Comment:
We'll probably use a technical user in future, therefore I suggest to parametrize all possible auth parameters, not only application credentials. Besides, if application credentials are not defined, this shouldn't cause an error.

Reply:
```
  listenAddr:
    http: :{{ .Values.nebula.port.http }} # default :1080
```